released/packages/coq-monae/coq-monae.0.2.2/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+
+homepage: "https://github.com/affeldt-aist/monae"
+dev-repo: "git+https://github.com/affeldt-aist/monae.git"
+bug-reports: "https://github.com/affeldt-aist/monae/issues"
+license: "LGPL-2.1-or-later"
+
+synopsis: "Monads and equational reasoning in Coq"
+description: """
+This Coq library contains a hierarchy of monads with their laws used
+in several examples of monadic equational reasoning."""
+
+build: [
+  [make "-j%{jobs}%"]
+  [make "sect5"]
+  [make "-C" "impredicative_set"]
+]
+install: [make "install"]
+depends: [
+  "coq" { (>= "8.11" & < "8.13~") | (= "dev") }
+  "coq-mathcomp-ssreflect" { (>= "1.11.0" & < "1.13~") }
```

Comment:
I think you could specify version bounds on the root package only, it can save you a lot of copy paste on updates.

Reply:
Doesn't it still make sense to specify bounds on analysis, infotheo, and paramcoq? I agree the version for the mathcomp packages that live in the same repo/archive as mathcomp-ssreflect could be viewed as redundant.

Reply:
Yes, I was only talking about the MC "main" packages.